lint/cmd/ccclint/ccclint.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"go/build"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/go-playground/errors/v5"
+	"github.com/google/uuid"
+)
+
+type ccclint struct {
+	pluginVersion string
+	verbose       bool
+}
+
+func (c *ccclint) run() error {
+	wd, err := os.Getwd()
+	if err != nil {
+		return errors.Wrap(err, "os.Getwd()")
+	}
+
+	cloneDir := filepath.Join(wd, uuid.New().String())
```

Comment:
We should use the standard lib instead of rolling our own.


```suggestion
	cloanDir, err := os.MkdirTemp("", "ccc-lint-")
	if err != nil {
		return errors.Wrap(err, "os.MkdirTemp()")
	}
    defer os.RemoveAll(cloanDir)
```